### ISSUE TITLE
Possible timezone-related bug.

### ISSUE THREAD

**Issue by sarg** [UniversaBlockchain/universa#9]
https://github.com/UniversaBlockchain/universa/blob/ca375a6ebe347d920161c2fdafe34a46c266f159/universa_core/src/main/java/com/icodici/universa/node/Elections.java#L380

currentTimeMillis() - системное время, которое не обязательно строго возрастает.
Например, может произойти скачок при синхронизации часов по ntp или при переходе на летнее время. Правильно использовать nanoTime

**Comment by sergeych** [comment]
оно не зависит от таймзоны и соответственно от летнего времени, и небольшие изменения в связи с работой nettime не должны как то влиять на логику - чуть увеличится предельное время до отмены выборов. nanoTime потенциально более дорогостоящий вызов, а его высокое разрешение нам не нужно.

**Comment by sarg** [comment]
Да, про таймзону я перепутал. Поинт про ntp не отменяется. Или просто, про перевод системных часов вручную.
